src/Doctrine/QueueableEntityNormalizer.php

```diff
@@ -51,6 +56,10 @@ public function supportsNormalization($data, $format = null)
      */
     public function denormalize($data, $class, $format = null, array $context = array())
     {
+        if (!array_key_exists('id', $data) || is_null($data['id'])) {
+            throw new InvalidArgumentException('Entity lacks of an id for deserialization');
```

Comment:
Dónde catcheamos esto sino estamos en la misma. Se queda muriendo la cola.

Reply:
now it will be handled on Worker::handleRejected because Job won't crash and instead return a null Job